Apps/PatientDataAccess/src/HealthData.cs

```diff
@@ -80,6 +80,32 @@ public record DiagnosticImagingExam : HealthData
         public bool? IsUpdated { get; set; }
     }
 
+    /// <summary>
+    /// The details of a BC Cancer screening exam.
+    /// </summary>
+    public record CancerScreeningExam : HealthData
+    {
+        /// <summary>
+        /// Gets or sets the cancer screening's event type.
+        /// </summary>
+        public CancerScreeningType EventType { get; set; }
+
+        /// <summary>
+        /// Gets or sets the cancer screening's program name.
+        /// </summary>
+        public string? ProgramName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the cancer screening's date.
+        /// </summary>
+        public DateTime EventTimestampUtc { get; set; }
+
+        /// <summary>
+        /// Gets or sets the cancer screening's result timestamp.
+        /// </summary>
+        public DateTime ResultTimestamp { get; set; }
```

Comment:
I'd suggest updating the documentation for all properties from "Gets or sets the cancer screening's X" to just "Gets or sets the X".